hyper-schema.json

```diff
@@ -69,9 +59,17 @@
         }
     },
     "definitions": {
+        "subSchema": {
+            "anyOf": [
+                {"$ref": "#"},
+                {"type": "boolean"},
+                {"$ref": "http://json-schema.org/draft/hyper-schema#/definitions/jsonReference"}
```

Comment:
I think it should be http://json-schema.org/draft/schema#/definitions/jsonReference